R/Rules-validity.R

```diff
@@ -330,6 +330,20 @@ v_increments_min <- function(object) {
   v$result()
 }
 
+#' @describeIn v_increments validates the [`IncrementsMaxToxProb`]
+v_increments_maxtoxprob <- function(object) {
+  v <- crmPack:::Validate()
+  v$check(
+    test_probabilities(object@prob),
+    "prob must be a vector of probabilities with minimum length 1 and no missing values"
+  )
```

Comment:
duplicate?

Reply:
`test_probabilities` doesn't let me test for a minimum length or (explicitly) for no minimum values.  So I used `test_numeric` for the latter but kept the error message the same.

Reply:
ah understand. Looking at https://github.com/openpharma/crmPack/blob/main/R/checkmate.R#L157, no need to check for no missing values, that is part of the check already. And the minimum length could be easily added there as another argument, and then avoid the overlapping code here

Reply:
Ok.  Will amend this PR and raise a new issue for the extra argument(s) for `check_probabilities()`.